packages/manager/modules/pci-object-storage/src/containers/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,37 @@
+{
+  "pci_projects_project_storages_containers_archive_title": "Cloud Archive",
+  "pci_projects_project_storages_containers_object_title": "Containers",
+  "pci_projects_project_storages_containers_object_description": "Hébergez votre catalogue de données dans notre solution Object Storage. Vous pouvez administrer ces données depuis votre application grâce aux API S3.",
+  "pci_projects_project_storages_containers_name_label": "Nom",
+  "pci_projects_project_storages_containers_region_label": "Localisation",
+  "pci_projects_project_storages_containers_id_label": "ID",
+
+  "pci_projects_project_storages_containers_offer_label": "Offre",
+  "pci_projects_project_storages_containers_public_label": "Public",
+  "pci_projects_project_storages_containers_toggle_public_succeed": "Votre container {{name}} est passé en statut Public.",
+  "pci_projects_project_storages_containers_toggle_private_succeed": "Le statut Public a été désactivé sur le container {{name}}.",
+  "pci_projects_project_storages_containers_toggle_fail": "Une erreur est survenue : {{message}}",
+
+  "pci_projects_project_storages_containers_containerType_label": "Type",
+  "pci_projects_project_storages_containers_containerType_public": "Public",
+  "pci_projects_project_storages_containers_containerType_private": "Privé",
+  "pci_projects_project_storages_containers_containerType_static": "Statique",
+
+  "pci_projects_project_storages_containers_storedObjects_label": "Nombre d'objets",
+  "pci_projects_project_storages_containers_storedBytes_label": "Espace utilisé",
+
+  "pci_projects_project_storages_containers_view_archive_label": "Voir les archives",
+  "pci_projects_project_storages_containers_view_object_label": "Voir les objets",
+  "pci_projects_project_storages_containers_view_add_user_label": "Ajouter un utilisateur à un conteneur",
+  "pci_projects_project_storages_containers_delete_label": "Supprimer",
+
+  "pci_projects_project_storages_blocks_add_archive_label": "Créer un conteneur d'archive",
+  "pci_projects_project_storages_blocks_add_object_label": "Créer un conteneur d'objets",
+  "pci_projects_project_storages_containers_offer_high_performance": "High Performance - S3",
+  "pci_projects_project_storages_containers_offer_standard_s3": "Standard - S3",
+  "pci_projects_project_storages_containers_offer_swift": "Standard - Swift",
+  "pci_projects_project_storages_containers_switch_to_private_label": "Passer en Privé",
+  "pci_projects_project_storages_containers_switch_to_public_label": "Passer en Public",
+  "pci_projects_project_storages_containers_swift_containers_load_error": "Une erreur est survenue sur un ou plusieurs de vos conteneurs Swift. Veuillez réessayer ultérieurement.",
+  "pci_projects_project_storages_containers_s3_containers_load_error": "Une erreur est survenue sur un ou plusieurs de vos conteneurs S3 dans la région {{ regions }}. Veuillez réessayer ultérieurement."
```

Comment:
`{{regions}}` means it's possible to have more than one region ? If yes the sentence should be adapted